src/properties.ts

```diff
@@ -33,15 +33,20 @@ import {
   SONAR_PROJECT_FILENAME,
 } from './constants';
 import { LogLevel, log } from './logging';
+import { getArch, getSupportedOS } from './platform';
 import { ScanOptions, ScannerProperties, ScannerProperty } from './types';
 
-const DEFAULT_PROPERTIES = {
-  [ScannerProperty.SonarUserHome]: path.join(
-    process.env.HOME ?? process.env.USERPROFILE ?? '',
-    SONAR_DIR_DEFAULT,
-  ),
-  [ScannerProperty.SonarScannerCliVersion]: SCANNER_CLI_VERSION,
-};
+function getDefaultProperties(): ScannerProperties {
```

Comment:
I modified this from a constant to a function otherwise we can't mock it (since the mock is applied after the constant is defined)